lib/src/formatter.dart

```diff
@@ -9,7 +9,8 @@ import 'hitmap.dart';
 
 abstract class Formatter {
```

Comment:
How about deprecating this entire class in favor of a static method? Is there anywhere we are taking `Formatter` or `FileHitMapsFormatter` as an argument instead of invoking the `format` method in the same place we decide which instance to construct?

Reply:
Done. I don't see anywhere that the base class is referenced at all.